### ISSUE TITLE
setup.sh: pip: no such option: --target /usr/local/lib/python3.7/dist-packages

### ISSUE THREAD

**Issue by Cnly** [pwndbg/pwndbg#875]
### Description

After #868, it seems that the `pip` invocation in `setup.sh` is broken. I got this error on Python 3.7 (and also Python 3.9)

```
no such option: --target /usr/local/lib/python3.7/dist-packages
```

when the following line is executed: https://github.com/pwndbg/pwndbg/blob/bf49bf8356dbe2a79e9e9fcbdef9ca409374d072/setup.sh#L151

It executed without problems when `${INSTALLFLAGS}` didn't have the quotes.

### Steps to reproduce

Just run `./setup.sh`.

### My setup

gdb: `GNU gdb (Debian 8.2.1-2+b3) 8.2.1`
Python: `3.7.3 (default, Jul 25 2020, 13:03:44) [GCC 8.3.0]`
pwndbg commit: `bf49bf8 (HEAD -> dev, origin/dev, origin/HEAD) Unit test fix (#868)`

**Comment by captain-woof** [comment]
I had the same issue, resolved it just now. The fix is easy. **The bug is just a string parsing issue**.

1. Open setup.sh
2. Search for `# Upgrade pip itself`
3. Replace `"${INSTALLFLAGS}"` by `${INSTALLFLAGS}` (remove the double quotation marks)
4. Do the same for the next line too.
5. Save the file, and run it.

**Comment by disconnect3d** [comment]
Fixed in #876
